vhdbuilder/packer/components.json

```diff
@@ -2,11 +2,9 @@
   "ContainerImages": [
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/autoscaler/addon-resizer:*",
-      "amd64OnlyVersions": [
-        "1.8.5"
-      ],
+      "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "1.8.14"
+        "1.8.18"
```

Comment:
should we just delete these? do any addons use them?

Reply:
These components are being referenced by addons (though different versions), so keeping the components for now.